generator/config/query/comment.yaml

```diff
@@ -11,3 +11,21 @@ arguments:
         name: comment
         type:
             - string
+tests:
+    -
+        name: 'Attach a Comment to an Aggregation Expression'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/query/comment/#attach-a-comment-to-an-aggregation-expression'
+        pipeline:
+            -
+                $match:
+                    x:
+                        $gt: 0
+                    $comment: 'Don''t allow negative inputs.'
```

Comment:
Noted that this syntax is specific to [Single-Quoted Style](https://yaml.org/spec/1.2.2/#single-quoted-style). Would it make more sense to use a [Double-Quoted Style](https://yaml.org/spec/1.2.2/#731-double-quoted-style) and avoid escaping altogether, or did you prefer single-quoted for consistency?

Reply:
I use single quotes everywhere to avoid risky parsing.